models/dhs_h3k27ac_test/feature_table.tsv

```diff
@@ -0,0 +1,41 @@
+feature	input_col	second_input	aggregate_function	fill_value	nice_name

```

Comment:
why is this directory labeled with 'test'? If it's not official, we shouldn't commit it